flowio/tests/flowdata_tests.py

```diff
@@ -25,9 +26,16 @@ def test_load_fcs_from_memory():
             mem_file = io.BytesIO(f.read())
             FlowData(mem_file)
 
+    def test_load_temp_file(self):
+        with tempfile.TemporaryFile() as tmpfile:
+            with open('examples/fcs_files/3FITC_4PE_004.fcs', 'r+b') as f:
+                shutil.copyfileobj(f, tmpfile)
+            tmpfile.seek(0)
+            out_data = FlowData(tmpfile)
```

Comment:
rename tmpfile -> tmp_file and close it after FlowData instance is created

Reply:
have added assert after the tempfile with clause (which should close it implicitly); let me know if you want an explicit close